docs/cli.md

```diff
@@ -0,0 +1,507 @@
+</table>
+
+### `rm`
+`rm` removes (deletes) a file. `rm -rf` removes a directory.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ rm euchre.cpp
+  $ rm -r stuff2/
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli013.png" class="invert-colors-in-dark-mode" alt="rm example" width="470px" />
+  </td>
+  </tr>
+</table>
+
+### `cd`
+`cd` changes directory.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ cd stuff/
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli014.png" class="invert-colors-in-dark-mode" alt="cd example" width="470px"/>
+  </td>
+  </tr>
+</table>
+
+### `mv`
+`mv` moves a file or directory into a different directory.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ mv main.cpp stuff/
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli005.png" class="invert-colors-in-dark-mode" alt="mv example" width="470px"/>
+  </td>
+  </tr>
+</table>
+
+`mv` is also used to rename a file or directory.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ mv example.txt new_name.txt
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli015.png" class="invert-colors-in-dark-mode" alt="mv rename example" width="470px"/>
+  </td>
+  </tr>
+</table>
+
+### `cp`
+`cp` copies a file.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ cp new_name.txt stuff/
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli016.png" class="invert-colors-in-dark-mode" alt="cp example"/>
+  </td>
+  </tr>
+</table>
+
+### `open` / `wslview`
+On macOS, `open` opens a file or directory with the default application, like a double click ([docs](https://ss64.com/osx/open.html)).
+
+On WSL (Windows), `wslview` opens a file or directory with the default application, like a double click ([docs](https://wslutiliti.es/wslu/man/wslview.html)).
+
+<video controls autoplay loop style="width: 100%; max-width: 640px; max-height: 480px;">
+  <source src="images/cli_vid002.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+<div class="primer-spec-callout warning" markdown="1">
+**WSL Pitfall:** You may need to install `wslu`, which includes `wslview`.
+```console
+$ sudo apt install wslu
+```
+</div>
+
+## Tips and Tricks
+
+### `clear` <kbd>Control</kbd> + <kbd>l</kbd>
+`clear` the terminal.  Pro-tip: <kbd>Control</kbd> + <kbd>l</kbd>.  That's a lowercase L.
+
+<video controls autoplay loop style="width: 100%; max-width: 640px; max-height: 480px;">
+  <source src="images/cli_vid003.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+### Tab complete <kbd>TAB</kbd>
+<kbd>TAB</kbd> autocompletes a file or directory name. It's faster than typing the whole thing.
+
+Type the first part of a filename, then press <kbd>TAB</kbd>.  Press again to show multiple completion options.
+```console
+$ cd ~/src/e  # Press TAB twice to see options
+eecs280/     eecs281/     eecs485/
+```
+
+<video controls autoplay loop style="width: 100%; max-width: 640px; max-height: 480px;">
```

Comment:
I'd like an example where there are multiple options.  WDYT about my `~/src/e` idea?  I'm open to other ideas, too.

Reply:
Great idea!